<launch>
    <node name="odom_path_node" pkg="zmcrobot_ros" type="odom_path" output="screen">
            <!-- param name="port" value="/dev/ttyACM0" / -->
            <param name="odom_topic" value="odom" />
            <param name="path_topic" value="odom_path" />
    </node>
  
    <node name="arduino_odom_path_node" pkg="zmcrobot_ros" type="odom_path" output="screen">
            <!-- param name="port" value="/dev/ttyACM0" / -->
            <param name="odom_topic" value="odom_arduino" />
            <param name="path_topic" value="odom_arduino_path" />
    </node>

</launch>
